infrastructure/restapi/src/main/java/tech/pegasys/teku/infrastructure/restapi/endpoints/JavalinRestApiRequest.java

```diff
@@ -86,6 +86,12 @@ public void respondError(final int statusCode, final String message)
     respond(statusCode, new HttpErrorResponse(statusCode, message));
   }
 
+  @Override
+  public Optional<String> getContentTypes() {
```

Comment:
Seems odd that this has a plural name but only returns 0 or 1 results.

Reply:
Also doesn't actually seem to be called anyway so maybe we can get rid of it?

Reply:
oh probably can, we did need it but i may have refactored that out...